src/meltano/core/plugin/singer/target.py

```diff
@@ -73,27 +73,31 @@ def writeline(self, line: str) -> None:
                 "Received state is invalid, incremental state has not been updated",
             )
```

Comment:
What do you think of this instead of the `if` statement:

```suggestion
            )
            return
```

Reply:
I had the same doubt about which one was better 😅. But I'm ok with return instead of if. Let me change it.

Reply:
Changed in https://github.com/meltano/meltano/pull/9258/commits/e6ee35e74b4822b281372cd7d7a8cfeea9ad227a